### DISCUSSION TITLE
Returns an error when publishing?

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#2593] [General]
NOTE MutpcAnorodbuid-tools folders exist choosing the latest verston (33.0.2) tor packaging

I publish for android, it gives such an error. I don't understand what's the matter.

**Comment by ajwfrost** [comment]
It should say (with the benefit of copy and paste):
`NOTE: Multiple Android build-tools folders exist, choosing the latest version (33.0.2) for packaging`

Basically that's just a note, not even a warning, it certainly shouldn't be treated as an error. It's just letting you know what the build tools are that will be used. If you have used Android Studio and their Android SDK manager for a while, you will likely have different sets of build tools installed. When the AIR Developer Tool is building an Android package it looks to see what build tools are available, and it will go through all available versions and choose what it thinks is the highest version number.

So that should be fine, it should still generate the APK/AAB file.

thanks
